iocage_lib/iocage.py

```diff
@@ -867,7 +867,7 @@ def exec(
         status, jid = self.list("jid", uuid=uuid)
 
         if not status and not start_jail:
-            if not ioc_common.is_tty():
+            if not INTERACTIVE:
```

Comment:
@skarekrow i think it would be better if we are explicit about this, it would work because when `ioc_common` is imported, the variable is set but it would still show up as an error on IDE's as it is unable to find a reference. I would recommend using `ioc_common.INTERACTIVE`